src/main/java/frc/robot/RobotContainer.java

```diff
@@ -120,54 +120,20 @@ private void controllerCheck() {
   private void initSubsystems() {
 
     // subsystems
-    for (Constants.Subsystems sub : Constants.Subsystems.values()) {
-      if (sub.isEnabled()) {
-
-        controllerCheck();
```

Comment:
controllerCheck sets axisCount1?

Need this for below, I believe.  Otherwise, looks good.